### ISSUE TITLE
[Error] Wasn't able to get URL

### ISSUE THREAD

**Issue by pythonnoob2** [5R33CH4/Python-Twitch-Viewbot#2]
I haven't changed anything within the code and have followed the steps.  Install all of the requirements, run the code with a # of threads but I receive "Wasn't able to get URL ***********Connection Error!*********

Any help would be appreciated.

**Comment by markito12** [comment]
i think twitch fixed, this repo its a copy of another repo

**Comment by pythonnoob2** [comment]
Ah shit, really? It looked similar to OhYou's code, but slightly different at the same time.  That;'s a bummer.

**Comment by tommaso-perondi** [comment]
Yes, this is copy of OhYou's code, it's not going to work, twitch fixed the method this code use (livestreamer), twitch now enfororces the clientid verification thus anonymous requests are no longer allowed and so bots like this can't work anymore.

**Comment by pythonnoob2** [comment]
I heard livestreamer was replaced by streamlink, any idea if that would work if we replaced it with that? or is Streamlink basically the same thing and will not work

**Comment by tommaso-perondi** [comment]
> I heard livestreamer was replaced by streamlink, any idea if that would work if we replaced it with that? or is Streamlink basically the same thing and will not work

i don't think so, the problem is the twitch api not the tool you use, they now enforce clientid verification so no more bots like this

**Comment by pythonnoob2** [comment]
oof.  Alright, thanks. I guess back to trying my luck with selenium. Which is a different method than this but isn't practical at all.

**Comment by optiix** [comment]
pythonnoob2

> 
> 
> oof. Alright, thanks. I guess back to trying my luck with selenium. Which is a different method than this but isn't practical at all.

Any luck. Tryinng to modify this script a little b4 I saw this posts... No luck yet. Probably right wbout Twitch fixed livestreamer. Solutions yet?

**Comment by wenosdias** [comment]
> pythonnoob2
> 
> > oof. Alright, thanks. I guess back to trying my luck with selenium. Which is a different method than this but isn't practical at all.
> 
> Any luck. Tryinng to modify this script a little b4 I saw this posts... No luck yet. Probably right wbout Twitch fixed livestreamer. Solutions yet?

Hi, have you found any solutions?

**Comment by AnonHexo** [comment]
no solution yet right?
